--- a/CMakeLists.txt	2020-09-30 16:35:45.060426301 -0400
+++ b/CMakeLists.txt	2020-09-30 16:49:45.520298594 -0400
@@ -1,25 +1,24 @@
 cmake_minimum_required(VERSION 3.12)
 project(ccm-ctrl VERSION 0.0.0)
 
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../daq-buildtools/cmake ${CMAKE_MODULE_PATH})
-include(DAQ)
+find_package(daq-buildtools)
 
 daq_setup_environment()
 
 find_package(appfwk REQUIRED)
 #find_package(ccm REQUIRED)      RS: ???
-#find_package(pistache REQUIRED) RS: ???
+find_package(Pistache REQUIRED)
 
 ##############################################################################
 daq_point_build_to( src )
 
-message("PISTACHE_LIB_DIR -> " $ENV{PISTACHE_LIB_DIR})
-message("PISTACHE_INC_DIR -> " $ENV{PISTACHE_INC_DIR})
+message("Pistache_LIBRARIES -> ${Pistache_LIBRARIES}")
+message("Pistache_INCLUDE_DIRS -> ${Pistache_INCLUDE_DIRS}")
 
 add_library(ccm-ctrl_restCommandFacility_duneCommandFacility src/RestCommandFacility.cpp src/RestEndpoint.cpp)
-target_include_directories (ccm-ctrl_restCommandFacility_duneCommandFacility PUBLIC $ENV{PISTACHE_INC_DIR})
+target_include_directories (ccm-ctrl_restCommandFacility_duneCommandFacility PUBLIC)
 target_link_libraries(ccm-ctrl_restCommandFacility_duneCommandFacility appfwk::appfwk 
-  $ENV{PISTACHE_LIB_DIR}/libpistache-0.0.002-git20200802.so
+  pistache_shared
 )
 
 ##############################################################################
